documentation/specs/src/economics/public-goods-funding.md

```diff
@@ -148,38 +151,37 @@ struct OnChainVote {
+- At epoch 54, the voting period ends and the votes are tallied. Since 80% > 66% of the voting power voted on this proposal (everyone except Charlie, who forgot to vote on her own proposal), the intitial condition is passed and the Proposal is active. Further, because out of the total votes, most were `Yay`, (75% > 50% threshold), the new Steward consisting of Bob and Charlie will be added to the Steward set. 
 
-- At epoch 54, the voting period ends and the votes are tallied. Since 80% > 33% of the voting power voted on this proposal (everyone except Charlie), the intitial condition is passed and the Proposal is active. Further, because out of the total votes, most were `Yay`, (75% > 50% threshold), a new council will be elected. The council that received the most votes, in this case `0xBobCharlieMultisig` is elected the new PGF council. The Council `(address: 0xElsaAndSisterMultisig, spending_cap: 50)` actually received 0 votes because Elsa's vote included the wrong spending_cap.
-
-- At epoch 57, Bob and Charlie have the effective power to carry out Public Goods Funding transactions.
+- At epoch 57, Bob and Charlie have the effective power to propose Public Goods Funding transactions (that may or may not be vetoed).
 ````
 
 ## Mechanism
 
-Once elected and instantiated, members of the PGF council will then unilaterally be able to propose and sign transactions for this purpose. The PGF council multisig will have an "allowance" to spend up to the `PGF` internal address's balance multiplied by the `spending_cap` variable.  Consensus on these transactions, in addition to motivation behind them will be handled off-chain, and should be recorded for the purposes of the "End of Term Summary".
+Once elected and instantiated, PGF Stewards will then unilaterally be able to sign transactions that propose either RPGF or CPGF funding. The PGF Stewards as a whole will have an "allowance" to spend up to the `PGF` internal address's balance.
+
+### Proposing Funding
+In order to propose funding, any Steward will be able to propose a PGFProposal through governance. Only Stewards will be valid authors of these proposals. There will be a minimum voting period set specifically for these types of proposals and can be changed by Governance. 
 
+This governance proposal will be such that it passes by default **unless** the following conditions are met:
 
-### PGF council transactions
-The PGF council members will be responsible for collecting signatures offline. One member will then be responsinble for submitting a transaction containing at least $k $ out of the signatures.
+Conditions to veto a PGF proposal:
+1. At least $\frac{1}{3}$ of voting power voted on the proposal
+2. Out of the votes, more than $50\%$ voted `Nay` on the proposal
+  - Further, if at least $\frac{2}{3}$ of voting power voted on the proposal, and the proposal was rejected, the Steward is removed from the set of stewards.
```

Comment:
hmm. one oddity with this is that if 51% vote against, if 1/3 vote the steward is not kicked out, but if 2/3 vote the steward is kicked out. maybe we should do 2/3 of 2/3? not sure.

Reply:
Yes I agree, 2/3 of 2/3 makes more sense

Reply:
Or it's slightly different now, I'm thinking 2/3 of voting-power must vote Nay. Lmk what you think, is this too strict?

Reply:
of the total voting power, or of the voting power voting?